### PR TITLE
interfaces: add dbusmenu, freedesktop and kde notifications to unity7 (Closes #1573188)

### PR DESCRIPTION
**Pull request by jdstrand**

### CODE REVIEW

**Comment by kyrofa** [comment]
My test case is the [Qt systray example snap]. This PR makes the indicator and notifications show up, so awesome there! However, the indicator has a menu with an option to quite the daemon, and selecting "quit" there doesn't actually quit the daemon-- you get:

```
May 13 12:54:24 xenial dbus[1823]: apparmor="DENIED" operation="dbus_method_call"  bus="session" path="/MenuBar" interface="com.canonical.dbusmenu" member="AboutToShowGroup" name=":1.20" mask="receive" pid=3386 label="snap.systray.systray" peer_pid=1966 peer_label="unconfined"
May 13 12:54:26 xenial dbus[1823]: apparmor="DENIED" operation="dbus_method_call"  bus="session" path="/MenuBar" interface="com.canonical.dbusmenu" member="EventGroup" name=":1.20" mask="receive" pid=3386 label="snap.systray.systray" peer_pid=1966 peer_label="unconfined"
```

Is that something that should be handled here?

**Comment by jdstrand** [comment]
@kyrofa oh I missed the indicator. I can confirm and I'll add an addition access to this commit.

File: interfaces/builtin/unity7.go [hunk 1]
```diff
@@ -145,13 +145,87 @@ dbus (receive,send)
...
+    path=/StatusNotifierItem
+    interface=org.freedesktop.DBus.Properties
+    member=Get*
+    peer=(label=unconfined),
+
+dbus (send)
+    bus=session
+    path=/org/freedesktop/Notifications
+    interface=org.freedesktop.Notifications
+    member="{GetServerInformation,Notify}"
```
Comment (3v1n0) [inline interfaces/builtin/unity7.go]: I think you also want GetCapabilities here. Clients might use that to behave differently (especially as unity notifications don't support "actions", while they have things such as "append")

**Comment by kyrofa** [comment]
@jdstrand with the latest commit this works perfectly for my test-case :+1: .

**Comment by zyga** [comment]
This looks good, thank you

File: interfaces/builtin/unity7.go (hunk 1 above)
Comment (jdstrand) [inline interfaces/builtin/unity7.go]: Thanks @3v1n0 , added.

**Comment by mvo5** [comment]
:+1: Thanks a lot @jdstrand

**Pull request merged**